apps/storefront-e2e/src/support/page_fragments/login.fragment.ts

```diff
@@ -9,7 +9,7 @@ export class LoginFragment {
   getLoginButton = () => this.getWrapper().contains('button', 'Login');
   getBEValidationError = () => this.getWrapper().find('oryx-notification');
 
-  login = (user: TestUserData) => {
+  login = (user: TestCustomerData) => {
```

Comment:
What is the actual difference in tests between user and customer?

Reply:
there is no difference now, but in the future, we might have "admin logged in as a customer" or something, so we should not use "user" in the code

I forget to change the variable name here, but I'll do it later, cause it might require more changes (this PR should be merge as soon as possible to unblock other branches)